errbot/backends/irc.py

```diff
@@ -566,7 +595,7 @@ def query_room(self, room):
         :returns:
             An instance of :class:`~IRCMUCRoom`.
         """
-        return IRCMUCRoom(room, bot=self)
+        return IRCMUCRoom(room, self)
```

Comment:
Again, while not necessarily invalid, keeping keyword arguments here would actually be safer (and more explicit, thus more clear)